sql/modules/Reconciliation.sql

```diff
@@ -93,6 +112,27 @@ $$Sets which lines of the report are cleared.$$;
 CREATE OR REPLACE FUNCTION reconciliation__delete_my_report(in_report_id int)
 RETURNS BOOL AS
 $$
+BEGIN
+    PERFORM approved FROM cr_report
+    WHERE id = in_report_id AND approved;
+    IF FOUND THEN
```

Comment:
Why do you introduce this? We have a trigger which blocks updates and deletes to approved reconciliation reports: `cr_report_block_changing_approved`.